src/main/java/me/zeroeightsix/kami/module/modules/movement/Flight.kt

```diff
@@ -6,20 +6,20 @@ import me.zeroeightsix.kami.util.EntityUtils
 import net.minecraft.client.Minecraft
 import net.minecraft.network.play.client.CPacketPlayer.PositionRotation
 
-/**
- * Created by 086 on 25/08/2017.
- */
 @Module.Info(
         name = "Flight",
         category = Module.Category.MOVEMENT,
         description = "Makes the player fly"
 )
-class Flight : Module() {
+object Flight : Module() {
     private val mode = register(Settings.enumBuilder(FlightMode::class.java).withName("Mode").withValue(FlightMode.VANILLA).build())
     private val speed = register(Settings.floatBuilder("Speed").withValue(10f).withMinimum(0f).build())
-    private val modifyGliding = register(Settings.booleanBuilder("ModifyGliding").withValue(true).withVisibility { mode.value == FlightMode.VANILLA }.build())
```

Comment:
why did we remove this option?

Reply:
Because we can just done it by checking if glide speed is 0

Reply:
![Screenshot_20200915_185747](https://user-images.githubusercontent.com/17222512/93273211-5a8b2180-f785-11ea-9d8a-6d9689824b56.png)
Well you forgot to do it here

Reply:
The motionY is supposed to 0 in static mode if no gliding 
![image](https://user-images.githubusercontent.com/62033805/93274022-49dbab00-f787-11ea-9fd6-f0b98af1d066.png)

Reply:
Oh, yea I can't do math.